compiler/modules/types/list.bal

```diff
@@ -255,7 +262,7 @@ function listInhabited(Context cx, FixedLengthArray members, SemType rest, ListC
             if listInhabited(cx, members, NEVER, neg.next) {
                 return true;
             }
-            foreach int i in len + 1 ..< negLen {
+            foreach int i in members.initial.length() + 1 ..< nt.members.initial.length() {
```

Comment:
Why are we changing this?

Reply:
This part of the code still had the large fixed-length array issue (https://github.com/ballerina-platform/nballerina/issues/820)